src/kbmod/image_collection.py

```diff
@@ -494,45 +596,252 @@ def get_standardizers(self, idxs=None, **kwargs):
         return [self.get_standardizer(idx, **kwargs) for idx in idxs]
 
     ########################
-    # FUNCTIONALITY (object operations, transformative functionality)
+    # IO
     ########################
-    def write(self, *args, format="ascii.ecsv", serialize_method=None, **kwargs):
+    @classmethod
+    def read(
+        cls, *args, format="ascii.ecsv", units=None, descriptions=None, unpack=True, validate=True, **kwargs
+    ):
+        """Create ImageCollection from a file containing serialized image
+        collection.
+
+        Parameters
+        ----------
+        *args : tuple, optional
+            Positional arguments passed through to data writer. If supplied the
+            first argument is the output filename.
+        format : `str`
+            File format specified, one of AstroPy IO formats that must support
+            comments.  Default: `ascii.ecsv`
+        units : `list`
+            List or dict of units to apply to columns.
+        descriptions : `list```
+            List or dict of descriptions to apply to columns
+        unpack : `bool`
+            If reading a packed image collection, unpack the shared values.
+        validate : `bool`
+            Validate that all required metadata exists.
+        kwargs: `dict`
+            Other keyword arguments passed onwards to AstroPy's `Table.read`.
+
+        Returns
+        -------
+        ic : `ImageCollection`
+            Image Collection
+        """
+        metadata = Table.read(*args, format=format, units=units, descriptions=descriptions, **kwargs)
+        if unpack and metadata.meta.get("is_packed", False):
```

Comment:
dict get method has a placeholder for a default return value. Otherwise None is returned by default. If is_packed is not in meta, False is now returned by default - i.e. the collection isn't packed.

Reply:
Good point. I missed that it was a `get()` from a dict.